src/worker/dump_db/dump-db.toml

```diff
@@ -216,3 +216,10 @@ published_by = "public"
 [versions_published_by.columns]
 version_id = "private"
 email = "private"
+
+[webhooks.columns]
+id = "public"
+owner_id = "public"
+webhook_url = "public"
+created_at = "public"
+updated_at = "public"
```

Comment:
I'm not sure we want to make any of this public. In particular the user provided URL could be sensitive.